docs/en/operations/configuration_files.md

```diff
@@ -1,10 +1,11 @@
 # Configuration Files {#configuration_files}
 
-The main server config file is `config.xml`. It resides in the `/etc/clickhouse-server/` directory.
+ClickHouse supports multi-file configuration management. The main server configuration file is `/etc/clickhouse-server/config.xml`. Other files must be in the `/etc/clickhouse-server/config.d` directory.
```

Comment:
* there's no examples with `config.d` below, while there's an example with `users.d` (actually this article needs way more examples)
* this is also worth mentioning:
```
% clickhouse-server --help 2>&1 | head -4
usage:
clickhouse-server [OPTION] [-- [ARG]...]
positional arguments can be used to rewrite config.xml properties, for
example, --http_port=8010
```
* probably we should name/explain this feature as "layered configuration" like in library that it's based on: https://pocoproject.org/docs/Poco.Util.LayeredConfiguration.html